plugins/Capacity.py

```diff
@@ -0,0 +1,133 @@
+from modules.OsmoseTranslation import T_
+from plugins.Plugin import Plugin
+from plugins.Plugin import TestPluginCommon
+
+###########################################################################
+##                                                                       ##
+## Copyrights Éric Gillet 2020                                           ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+
+class Capacity(Plugin):
+    def init(self, logger):
+        Plugin.init(self, logger)
+
+        self.errors[35001] = self.def_class(
+            item=3500,
+            level=2,
+            tags=["tag"],
+            title=T_("Invalid capacity value"),
+            detail=T_("""A capacity tag value is incorrect."""),
+            resource="https://wiki.openstreetmap.org/wiki/Key:capacity for more information",
+        )
+
+    def node(self, data, tags):
+        try:
+            if tags["capacity"] == "":
+                # Ignore errors that should be reported by generic prefix analysers
+                return
+            total_capacity = int(tags["capacity"])
+        except KeyError:
+            # No capacity tag
+            return
```

Comment:
Could you just check for "capacity" in tags, rather than raisin an exception (to be the same as other plugins).

Reply:
Changed in be3f47bc.